glide.lock

```diff
@@ -28,17 +28,19 @@ imports:
 - name: github.com/mailru/easyjson
   version: 9d6630dc8c577b56cb9687a9cf9e8578aca7298a
   subpackages:
-  - bootstrap
```

Comment:
Changed from alphabetical to not alphabetical :(

Not sure what tool is doing this...